### PR TITLE
Update ode_solve.md

### PR DESCRIPTION
**Pull request by gstein3m**

### CODE REVIEW

File: docs/src/solvers/ode_solve.md [hunk 1]
```diff
@@ -498,6 +498,10 @@ These methods also have option `nlsolve` same as SDIRK methods. These methods al
   stiff-aware 3rd order interpolant. 4th order on linear parabolic problems and
   3rd order accurate on nonlinear parabolic problems (as opposed to lower if not
   corrected).
+- `Rodas4P2` - A 4th order A-stable stiffly stable Rosenbrock method with a
+  stiff-aware 3rd order interpolant. 4th order on linear parabolic problems and
+  3rd order accurate on nonlinear parabolic problems. It is an improvement of Roadas4P 
+  and in case of inexact Jacobians a second order W method.
```
Comment (ChrisRackauckas) [inline docs/src/solvers/ode_solve.md:504]: oh cool, didn't realize it was also a W method. This should get some good benchmarks

File: docs/src/solvers/ode_solve.md [hunk 2]
```diff
@@ -498,6 +498,10 @@ These methods also have option `nlsolve` same as SDIRK methods. These methods al
   stiff-aware 3rd order interpolant. 4th order on linear parabolic problems and
   3rd order accurate on nonlinear parabolic problems (as opposed to lower if not
   corrected).
+- `Rodas4P2` - A 4th order A-stable stiffly stable Rosenbrock method with a
```
Comment (ChrisRackauckas) [inline docs/src/solvers/ode_solve.md:501]: No L-stability?

File: docs/src/solvers/ode_solve.md [hunk 3]
```diff
@@ -498,6 +498,10 @@ These methods also have option `nlsolve` same as SDIRK methods. These methods al
...
+  stiff-aware 3rd order interpolant. 4th order on linear parabolic problems and
```
Comment (ChrisRackauckas) [inline docs/src/solvers/ode_solve.md:502]: you didn't implement this interpolation.

Comment (ChrisRackauckas) [reply docs/src/solvers/ode_solve.md:502]: nevermind, you did.

File: docs/src/solvers/ode_solve.md (hunk 2 above)
Comment (ChrisRackauckas) [reply docs/src/solvers/ode_solve.md]: saw in the paper it's L-stable so I updated it.

**Comment by ChrisRackauckas** [comment]
I took a look at the paper, saw it was L-stable and an Index-1 DAE solver and added it to the DAE list. Also just noticed you are the author, awesome!

Want to give it a spin in https://github.com/SciML/SciMLBenchmarks.jl ? See https://github.com/SciML/SciMLBenchmarks.jl#contributing.

 https://benchmarks.sciml.ai/html/StiffODE/ROBER.html might be a good first one to try. Try it locally first using the code in:

https://github.com/SciML/SciMLBenchmarks.jl/blob/master/benchmarks/StiffODE/ROBER.jmd

Our automated benchmark updating system is down right now so you can PR it but we won't be able to rebuild the websites for another week. But check locally how it does. I would just test the higher accuracy benchmarks:

https://github.com/SciML/SciMLBenchmarks.jl/blob/master/benchmarks/StiffODE/ROBER.jmd#L223

I am curious how it fares vs Rodas5. In fact, I would be interested in collaborating with you in possibly doing some tableau optimization to Rodas5: there's only one tableau and I feel it's not all that great.

**Comment by gstein3m** [comment]
Yes, I will perform some benchmarks locally.

Regarding optimizing rodas5, this is a bigger task, but I would like to consider it in future.


Best regards


Gerd

------------------------------------------------------------------------------------
________________________________
Von: Christopher Rackauckas <notifications@github.com>
Gesendet: Mittwoch, 13. Januar 2021 18:51
An: SciML/DiffEqDocs.jl
Cc: Steinebach, Gerd; Author
Betreff: Re: [SciML/DiffEqDocs.jl] Update ode_solve.md (#429)


I took a look at the paper, saw it was L-stable and an Index-1 DAE solver and added it to the DAE list. Also just noticed you are the author, awesome!

Want to give it a spin in https://github.com/SciML/SciMLBenchmarks.jl ? See https://github.com/SciML/SciMLBenchmarks.jl#contributing.

https://benchmarks.sciml.ai/html/StiffODE/ROBER.html might be a good first one to try. Try it locally first using the code in:

https://github.com/SciML/SciMLBenchmarks.jl/blob/master/benchmarks/StiffODE/ROBER.jmd

Our automated benchmark updating system is down right now so you can PR it but we won't be able to rebuild the websites for another week. But check locally how it does. I would just test the higher accuracy benchmarks:

https://github.com/SciML/SciMLBenchmarks.jl/blob/master/benchmarks/StiffODE/ROBER.jmd#L223

I am curious how it fares vs Rodas5. In fact, I would be interested in collaborating with you in possibly doing some tableau optimization to Rodas5: there's only one tableau and I feel it's not all that great.

-

**Pull request merged**